l10n_it_fatturapa_in/migrations/16.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,10 @@
+#  Copyright 2023 Nextev Srl
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    view = env.ref("l10n_it_fatturapa_in.view_fatturapa_in_attachment_form")
```

Comment:
Io piuttosto mergerei https://github.com/OCA/l10n-italy/pull/3785 e in generale allinerei i sorgenti della 14 alla 16 quando sono più recenti, invece di disfare le aggiunte con script di migrazione.